packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -728,20 +728,23 @@ export function createLodestarMetrics(
         buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
       }),
     },
-    recoverDataColumnSidecars: {
-      elapsedTimeTillReconstructed: register.histogram({
-        name: "lodestar_data_column_sidecar_elapsed_time_till_reconstructed_seconds",
-        help: "Time elapsed between block slot time and the time data column sidecar reconstructed",
-        buckets: [2, 4, 6, 8, 10, 12],
+    gossipDataColumn: {
+      recvToValidation: register.histogram({
+        name: "lodestar_gossip_data_column_received_to_gossip_validate",
+        help: "Time elapsed between data column received and data column validation",
+        buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
       }),
-      custodyBeforeReconstruction: register.gauge({
-        name: "lodestar_data_columns_in_custody_before_reconstruction",
-        help: "Number of data columns in custody before reconstruction",
+      validationTime: register.histogram({
+        name: "lodestar_gossip_data_column_gossip_validate_time",
+        help: "Time elapsed for data column validation",
+        buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
       }),
-      reconstructionResult: register.gauge<{result: RecoverResult}>({
-        name: "lodestar_data_column_sidecars_reconstruction_result",
-        help: "Data column sidecars reconstruction result",
-        labelNames: ["result"],
+    },
+    blobs: {
+      bySource: register.gauge<{source: BlobsSource}>({
```

Comment:
this metrics seems to be a counter
```suggestion
      bySource: register.counter<{source: BlobsSource}>({
```